### PR TITLE
Latest

### PR DESCRIPTION
**Pull request by SoSeductive**

### CODE REVIEW

File: changes.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### 2.4.6 ( December 21, 2021 )
+* Tested with latest WordPress version.
```
Comment (planv) [inline changes.md]: * Added compatibility with WordPress 5.8.2.

File: readme.txt [hunk 2]
```diff
@@ -207,6 +207,9 @@ If you submit a popular idea on UserVoice, we WILL integrate it sooner or later.
 
 == Upgrade Notice ==
 
+= Version 2.4.6 =
+* Tested with WordPress v5.8.2.
```
Comment (planv) [inline readme.txt]: Remove this block from `Upgrade Notice`:

```
= Version 2.4.5 =
* Fixed issue with validating image in WP Property FEPS addon.

= Version 2.4.3 =
* Remove Support tab.

= Version 2.4.2 =
* Added compatibility with WordPress 5.6.

= Version 2.4.1 =
* Added compatibility with WordPress 5.5.
```

File: readme.txt [hunk 3]
```diff
@@ -239,6 +242,9 @@ If you submit a popular idea on UserVoice, we WILL integrate it sooner or later.
 
 == Changelog ==
 
+= 2.4.6 =
+* Tested with WordPress v5.8.2.
```
Comment (planv) [inline readme.txt]: * Added compatibility with WordPress 5.8.2.

**Comment by SoSeductive** [comment]
@planv Updated

File: readme.txt (hunk 3 above)
Comment (SoSeductive) [reply readme.txt]: @planv Fixed

File: readme.txt (hunk 2 above)
Comment (SoSeductive) [reply readme.txt]: @planv Fixed

File: changes.md (hunk 1 above)
Comment (SoSeductive) [reply changes.md]: @planv Fixed

File: readme.txt (hunk 2 above)
Comment (SoSeductive) [reply readme.txt]: @planv Updated

File: readme.txt [hunk 4]
```diff
@@ -207,17 +207,8 @@ If you submit a popular idea on UserVoice, we WILL integrate it sooner or later.
...
-
-= Version 2.4.3 =
-* Remove Support tab.
-
-= Version 2.4.2 =
-* Added compatibility with WordPress 5.6.
-
-= Version 2.4.1 =
-* Added compatibility with WordPress 5.5.
+= Version 2.4.6 =
```
Comment (planv) [inline readme.txt]: Remove it as well

```
= Version 2.4.6 =
* Added compatibility with WordPress 5.8.2.
```

Comment (SoSeductive) [reply readme.txt]: @planv Done

**Review by planv — APPROVED** [review]
looks good

**Pull request merged**